<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Empleados</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sells_tab">
      <attribute name="title">
       <string>Venta</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Venta en proceso</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="3">
           <widget class="QPushButton" name="sell_cancel_button">
            <property name="text">
             <string>Cancelar venta</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="sell_total_label">
            <property name="text">
             <string>Total: $0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="sell_add_product">
            <property name="text">
             <string>Agregar componente</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="sell_delete_product">
            <property name="text">
             <string>Eliminar componente</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QTableWidget" name="sell_table_widget">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="sell_finale_button">
            <property name="text">
             <string>Terminar venta</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>empty</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cliente:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="client_comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="warranty_tab">
      <attribute name="title">
       <string>Garantía</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Añadir garantía</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QTableWidget" name="warranty_table"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Los productos que siguen en la venta se mostrarán.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="warranty_add_button">
            <property name="text">
             <string>Dar garantía</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="client_tab">
      <attribute name="title">
       <string>Cliente</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="add_pushButton">
           <property name="text">
            <string>Añadir cliente</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="discard_pushButton">
           <property name="text">
            <string>Descartar datos</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Cliente</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="3" column="1">
           <widget class="QLineEdit" name="rut_lineEdit"/>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="methodpay_comboBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Rut:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Metodo de pago:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Apellido:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Direccion:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="address_lineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="name_lineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lastname_lineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Telefono:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="phone_lineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Correo:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="mail_lineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
